### ISSUE TITLE
Unmatched " character terminating declaration causes following rules to be swallowed

### ISSUE THREAD

**Issue by pocketjoso** [csstree/csstree#93]
Hi @lahmatiy, how are you doing? Thanks again for this great library.

I just came across a case where csstree never recovers after a syntax errors, whereas browsers do. Ideally for me csstree could handle this - but not sure how easy it would be for you to handle.

## Reduced test case
```css

.first {
  color: red"
}
.thisAndAllFollowingRulesGetSwallowed {
  color: red
}
```
You can see this in effect in the [astexplorer](https://astexplorer.net/#/gist/aaf18a12efd85c47481e06638f79f404/d07ccbd311e549898715dc290966c994771e0756).

## expectation
The AST parsing recovers, and correctly parses rules following the broken `.first` one.

Cheers,
Jonas

**Comment by lahmatiy** [comment]
Hi @pocketjoso and thank you!
Yeah, that's a bug. Unescaped newline must terminate a string and produce a `<bad-string-token>`. I knew about this, but issue were too rare so I didn't fix it up. Will do now, thank you for the issue.
To fix this we need add bad-string-token to tokenizer. I worked on [new tokenizer](https://github.com/csstree/csstree/tree/rework-tokenizer) for a while and planned to add this. Unfortunately, I didn't finished that work and don't know when I can (busy on another projects). However your support can help :)

**Comment by lahmatiy** [comment]
Fixed by commit 0758b2d2b6e23afd452aba6143cb55f86b30908e which implements `<bad-string-token>`
